doc/ale.txt

```diff
@@ -2318,6 +2318,12 @@ ale#linter#Define(filetype, linter)                       *ale#linter#Define()*
                          `type` - Defaults to `'E'`.
                          `nr` - Defaults to `-1`.
 
+                           Numeric error code. If `nr` is not `-1`, `code` should have
```

Comment:
oh, and I should change this to say something about being cast to a string.

Reply:
Say that the code is required to be a String. This is documentation for people looking up which keys they can export, and what the values should look like.